relay/sources/relayd/src/remote_run.rs

```diff
@@ -65,6 +65,20 @@ impl RemoteRun {
         })
     }
 
+    pub fn reply_keep_output(&self, job_config: Arc<JobConfig>) -> impl warp::reply::Reply
+    {
+        if self.run_parameters.keep_output == true
```

Comment:
no need to explicitely compare to `true`, `cargo clippy` would likely mention it